palette/legend.go

```diff
@@ -0,0 +1,118 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package palette
```

Comment:
I don't understand why this is in palette. It seems more like something for plotter - and then would be a separate PR.